Sources/SkipUI/SkipUI/Components/ProgressView.swift

```diff
@@ -95,7 +95,13 @@ public struct ProgressView : View {
         if value == nil || total == nil {
             LinearProgressIndicator(modifier: modifier, color: color)
         } else {
-            LinearProgressIndicator(progress: Float(value! / total!), modifier: modifier, color: color)
+            LinearProgressIndicator(
+                progress: { Float(value! / total!) },
```

Comment:
Thanks for the deprecated API fix. I'd like to leave everything else as using default values (i.e. just exclude the gapSize and drawStopIndicator parameters) for the normal Compose look. We can eventually add custom modifiers to change the Material view, as we've done for other components.

And could you also make the fix for CircularProgressIndicator?